bin/anthology/papers.py

```diff
@@ -90,7 +101,7 @@ def from_xml(xml_element, *args):
             paper.attrib['revision'].insert(0, {
                 "value": "{}v1".format(paper.full_id),
                 "id": "1",
-                "url": data.ANTHOLOGY_URL.format( "{}v1".format(paper.full_id)) } )
+                "url": data.ANTHOLOGY_URL.format( "{}v1.pdf".format(paper.full_id)) } )
```

Comment:
Should this be `data.ANTHOLOGY_PDF.format(f"{paper.full_id}v1")` maybe?

Reply:
Yes, obviously! Thank you!